indra/llcharacter/llcharacter.h

```diff
@@ -276,7 +276,6 @@ class LLCharacter
 	U32					mSkeletonSerialNum;
 	LLAnimPauseRequest	mPauseRequest;
 
-    virtual void        postMotionUpdate() {}
```

Comment:
Rather than create a virtual `postMotionUpdate()` method here with no-op default we make `updateMotions()` virtual and override that in the one base class that needs it.